miden-lib/asm/kernels/transaction/lib/prologue.masm

```diff
@@ -9,6 +9,12 @@ use.kernel::constants
 use.kernel::memory
 use.kernel::utils
 
+# CONSTS
+# =================================================================================================
+
+# Max U32 value, used for initializing the expiration block delta
+const.U32_MAX=0xFFFFFFFF
```

Comment:
Thinking a bit more about it, I'm not sure it would be the delta that we store in the bookkeeping section but the absolute expiration block number.

Reply:
If set by the user, then it's the relative delta. `u32::MAX` acts as a sort of the default value for that, I'd say. We could make it so that we store the actual absolute expiration block number by summing the input and the block ref number every time the script sets the expiration delta.

Reply:
Yeah - that's how I was thinking about it. We store the absolute expiration block height in the prologue and then in `set_expiration_delta` we apply the relative delta to it. The work in the epilogue is then minimal (we just need to read the absolute value from memory and put it onto the stack).

I haven't thought it through completely - but seems like it would end up a bit cleaner.

Also, thinking more about it, i'm not sure limiting the delta to 256 has good motivation. Maybe we should bump it up to something like `u16::MAX`?

Reply:
Yeah, agreed that 256 is probably too low. Bumped it up.